lib/commands/context.js

```diff
@@ -699,6 +703,24 @@ function isUrlIgnored(url, safariIgnoreWebHostnames) {
   return false;
 }
 
+/**
+ * https://github.com/appium/appium/issues/20741
+ *
+ * @this {XCUITestDriver}
+ * @returns {Promise<void>}
+ */
+export async function notifyBiDiContextChange() {
+  const name = await this.getCurrentContext();
+  const type = name === NATIVE_WIN ? 'NATIVE' : 'WEB';
+  this.eventEmitter.emit(BIDI_EVENT_NAME, {
+    method: 'appium.contextUpdate',
```

Comment:
Not for this pr itself, but maybe it would be nice to have a central definition file for BiDi methods and get this value from the file as a const. We already have log's method name so it will be also the same.

The main purpsoe is to maintain method names in one place (and understand what kind of methods are curretly supported by this driver)

Reply:
Good idea. I have refactored BiDi event models in this PR.